crates/eframe/src/web/events.rs

```diff
@@ -402,6 +402,22 @@ pub fn install_canvas_events(runner_container: &mut AppRunnerContainer) -> Resul
         &canvas,
         "wheel",
         |event: web_sys::WheelEvent, mut runner_lock| {
+            {
+                let unit = match event.delta_mode() {
+                    web_sys::WheelEvent::DOM_DELTA_PIXEL => egui::MouseWheelUnit::Point,
+                    web_sys::WheelEvent::DOM_DELTA_LINE => egui::MouseWheelUnit::Line,
+                    web_sys::WheelEvent::DOM_DELTA_PAGE => egui::MouseWheelUnit::Page,
+                    3u32..=u32::MAX => return,
+                };
+                let delta = egui::vec2(event.delta_x() as f32, event.delta_y() as f32);
```

Comment:
further down in this code we flip the sign of the delta to match that of winit - don't you think we should do that here too?

Please test your code on both native and web to make sure the signs are the same!

Reply:
That's actually what I was asking about in the first comment in this PR. There are multiple differences between targets, and if egui changes only some of them, users still have to be aware of the others, which might make it even more confusing. And if we change them all, then it just becomes a duplicate of the `Scroll` event (but without accounting for the modifier keys). That could be a choice, but I wasn't sure of where to draw the line. Do I just flip the sign of the delta, or do I handle the other differences as well?

Reply:
Without the signs being correct it is almost useless imho.

A common use-case for this is to catch a single step of the scroll-wheel. The magnitude of this is different on different platforms, but as long as the sign is the same a user can still do `let change = delta.y.sign();` to catch the discreet scroll-tick events.

Reply:
I see your point. Although I guess you would have to do `let change = (delta.x + delta.y).sign();`, because of the macOS thing as mentioned down below in the code. But in any case, this seems like a good compromise. I changed it.

I also noticed that the touchpad gesture is also returned as a wheel event by the backend, but it uses the unit `Point` on native and `Line` on web, both with about the same delta for the same speed. I don't know if that is worth dealing with as well. At least the sign is the same, though, and this already affects the `Scroll` event anyway. Scrolling with the touchpad on web goes much faster than on native.